### ISSUE TITLE
[ Google Cloud Platform (GCP) Compute metrics ] TSDB enablement

### ISSUE THREAD

**Issue by agithomas** [elastic/integrations#6711]
## Test Environment Setup
- [x] Creation of Test Environment

## Datastream : `Compute Metrics` 
  - [ ] Add dimension fields - https://github.com/elastic/integrations/pull/6781 
  - [ ] Add metric_type mappings - https://github.com/elastic/integrations/pull/6798

#### Verification and validation
  - [ ] Verification of data in visualisation after enabling the TSDB flag in kibana
  - [ ] Verification of the count of documents (before & after TSDB enablement) in Discover Interface
  - [ ] Verify if field mapping is correct in the data stream template.
  - [ ] Verify if the TSDB can be disabled using the API method
  - [ ] Verify if the TSDB can be re-enabled after 4 hours, following disablement using API method, making use of package upgrade. 
  - [ ] Add screenshots of dashboards (Recommended), if applicable. 
 
## TSDB Release PR
 - [ ] PR to enable TSDB
 - [ ] Update the tracking sheet (GA release version, github link, status)

## Blocked By 
- https://github.com/elastic/integrations/issues/6568

**Comment by lalit-satapathy** [comment]
@agithomas, @gpop63,

Do we still need this separate issue?  Can we keep [this](https://github.com/elastic/integrations/issues/7555) gcp issue up-to-date for TSDB status?
